x/incentives/keeper/distribute.go

```diff
@@ -343,19 +371,46 @@ func (k Keeper) Distribute(ctx sdk.Context, gauges []types.Gauge) (sdk.Coins, er
 	distrInfo := newDistributionInfo()
 
 	locksByDenomCache := make(map[string][]lockuptypes.PeriodLock)
-	totalDistributedCoins := sdk.Coins{}
+	totalDistributedCoins := sdk.NewCoins()
+
+	// get pool Id corresponding to the gaugeId
+	incentiveParams := k.GetParams(ctx).DistrEpochIdentifier
+	currentEpoch := k.ek.GetEpochInfo(ctx, incentiveParams)
+
 	for _, gauge := range gauges {
-		filteredLocks := k.getDistributeToBaseLocks(ctx, gauge, locksByDenomCache)
-		// send based on synthetic lockup coins if it's distributing to synthetic lockups
 		var gaugeDistributedCoins sdk.Coins
-		var err error
-		if lockuptypes.IsSyntheticDenom(gauge.DistributeTo.Denom) {
-			gaugeDistributedCoins, err = k.distributeSyntheticInternal(ctx, gauge, filteredLocks, &distrInfo)
+		pool, isCLPool := k.GetValidConcentratedLiquidityGauge(ctx, gauge.Id, currentEpoch.Duration)
+		// only want to run this logic if the gaugeId is associated with CL PoolId
+		if isCLPool {
+			for _, coin := range gauge.Coins {
+				emissionRate := sdk.NewDecFromInt(coin.Amount).QuoTruncate(sdk.NewDec(currentEpoch.Duration.Milliseconds()).QuoInt(sdk.NewInt(1000)))
```

Comment:
I think an in line comment explaining the calculation for the emission rate would be appropriate here.